sdk/communication/azure-communication-callingserver/README.md

```diff
@@ -0,0 +1,215 @@
+```java
+CommunicationIdentifier source = new CommunicationUserIdentifier("<acs-user-identity>");
+
+List<CommunicationIdentifier> targets = new ArrayList<>();
+
+CommunicationIdentifier firstCallee = new CommunicationUserIdentifier("<acs-user-identity-1>");
+CommunicationIdentifier secondCallee = new CommunicationUserIdentifier("<acs-user-identity-2>");
+
+targets.add(firstCallee);
+targets.add(secondCallee);
+
+String callbackUri = "<callback-uri-for-notification>";
+
+List<CallModality> requestedModalities = new ArrayList<>();
+requestedModalities.add(CallModality.AUDIO);
+requestedModalities.add(CallModality.VIDEO);
+
+List<EventSubscriptionType> requestedCallEvents = new ArrayList<>();
+requestedCallEvents.add(EventSubscriptionType.DTMF_RECEIVED);
+requestedCallEvents.add(EventSubscriptionType.PARTICIPANTS_UPDATED);
+
+CreateCallOptions createCallOptions = new CreateCallOptions(
+    callbackUri,
+    requestedModalities,
+    requestedCallEvents);
+
+CreateCallResponse createCallResult =  callClient.createCall(source, targets, createCallOptions);
+
+String callId = createCallResult.getCallLegId();
+```
+
+Add a participant to a Call:
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ReadmeSamples.java#L109-L110 -->
+```java
+CommunicationIdentifier thirdCallee = new CommunicationUserIdentifier("<acs-user-identity-2>");
+callClient.addParticipant(callId, thirdCallee, "ACS User 2", "<string-for-tracing-responses>");
+```
+
+Hangup a Call:
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ReadmeSamples.java#L89-L89 -->
+```java
+callClient.hangupCall(callId);
+```
+
+Delete a Call:
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ReadmeSamples.java#L99-L99 -->
+```java
+callClient.deleteCall(callId);
+```
+
+### Start, Stop, Pause, Resume, and Get a recording
+
+Create a ConverationClient: 
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ConversationClientReadmeSamples.java#L27-L36 -->
+```java
+String endpoint = "https://<RESOURCE_NAME>.communcationservices.azure.com";
+
+// Your connectionString retrieved from your Azure Communication Service
+String connectionString = "https://<resource-name>.communication.azure.com/;<access-key>";
+
+// Initialize the call client
+final ConversationClientBuilder builder = new ConversationClientBuilder();
+builder.endpoint(endpoint)
+    .connectionString(connectionString);
+ConversationClient conversationClient = builder.buildClient();
+```
+
+Start a Recording: 
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ConversationClientReadmeSamples.java#L48-L51 -->
+```java
+String conversationId = "<conversationId recieved from starting call>";
+String recordingStateCallbackUri = "<webhook endpoint to which calling service can report status>";
+StartCallRecordingResponse response = conversationClient.startRecording(conversationId, recordingStateCallbackUri);
+String recordingId = response.getRecordingId();
```

Comment:
Why do we need like 140? Let's try to keep the samples as simple as possible

Reply:
Line removed